packages/ai-bot/helpers.ts

```diff
@@ -681,14 +681,24 @@ Attached files:
 ${attachedFilesToPrompt(attachedFiles)}
 `;
 
+  let cardPatchTool = tools.find((tool) => tool.function.name === 'patchCard');
+  let sourceCodeEditingSkillCard = skillCards.find(
+    (card) => card.attributes?.title === 'Source Code Editing',
+  );
+
+  if (cardPatchTool && sourceCodeEditingSkillCard) {
+    systemMessage += `First determine if the user's request involves template editing, visual changes, or styling, or card logic changes.
```

Comment:
Based on my experiments it's far more reliable if we put this message at the beginning, before skill prompts are added.

Reply:
I don't really like the idea of special casing some skills in the aibot.

Reply:
Right, as discussed I think renaming the ambiguous `patchCard` could be beneficial in a way where we won't have to do that.